tracer/samples/AutomaticTraceIdInjection/Log4NetExample/Log4NetExample.csproj

```diff
@@ -7,7 +7,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Datadog.Trace.Bundle" Version="2.42.0" />
+    <PackageReference Include="Datadog.Trace.Bundle" Version="2.43.0" />
```

Comment:
This might be linked to my question for the dockerfile:
wondering if it should be `3.0.0` instead ?

Reply:
So these ones are meant to point to an already existing package that we can download, so no, they shouldn't be 3.0.0.

But... 2.43 doesn't exist either, so this is also wrong. Basically the automation doesn't quite work correctly here. Given we may want to discuss more major versions in the future, I think I'll create a new GitHub action and try this again instead!

Reply:
oh I see. Thx :)